test/irb/test_irb.rb

```diff
@@ -963,4 +963,34 @@ def bar
       end
     end
   end
+
+  class ContextModeTest < TestIRB::IntegrationTestCase
+    # RUBY_BOX=1 may need more time to start IRB
+    TIMEOUT_SEC = TestIRB::IntegrationTestCase::TIMEOUT_SEC + 5
```

Comment:
I think this can be 
```suggestion
    TIMEOUT_SEC = self::TIMEOUT_SEC + 5
```

```rb
class Foo
  TIMEOUT = 10
end

class Bar < Foo
  TIMEOUT = self::TIMEOUT + 10
end

puts Bar::TIMEOUT # 20
```

Reply:
Actually, it can be even shorter
```ruby
TIMEOUT = TIMEOUT + 5
```
or 
```ruby
TIMEOUT += 5
```
but the visual of the code, especially the last one, looks like a constant reassignment. (It's not, though)
So my code is... a bit redundant but I'd like to use the redundant form here.